src/main/kotlin/lotto/domin/MatchCount.kt

```diff
@@ -0,0 +1,6 @@
+package lotto.domin
+
+class MatchCount(
```

Comment:
```suggestion
data class MatchCount(
```
데이터를 관리하는 클래스라면, data class를 활용해보면 어떨까요?